Firestore/core/test/unit/local/remote_document_cache_test.h

```diff
@@ -53,10 +54,18 @@ class RemoteDocumentCacheTest : public ::testing::TestWithParam<FactoryFunc> {
   RemoteDocumentCacheTest();
 
  protected:
-  model::Document SetTestDocument(absl::string_view path);
-  model::Document SetTestDocument(absl::string_view path,
-                                  int update_time,
-                                  int read_time);
+  model::MutableDocument SetTestDocument(absl::string_view path);
+  model::MutableDocument SetTestDocument(absl::string_view path,
+                                         google_firestore_v1_Value data);
+  model::MutableDocument SetTestDocument(absl::string_view path,
+                                         int update_time,
+                                         int read_time);
+  model::MutableDocument SetTestDocument(absl::string_view path,
+                                         google_firestore_v1_Value data,
+                                         int update_time,
+                                         int read_time);
+  void VerifyValue(model::MutableDocument actual_doc,
```

Comment:
Nit: pass `actual_doc` by const reference?

Reply:
Not used anymore